src/workers/index.ts

```diff
@@ -40,6 +42,8 @@ export const workers: Worker[] = [
   deleteUserFromMailingList,
   commentUpvotedRep,
   commentUpvoteCanceledRep,
+  commentMarkdownImages,
```

Comment:
I think you miss some workers here. I see more new subscriptions then workers here

Reply:
Yes, I thought we can put 2 workers (edit and delete) in a single subscription. I am currently adding a dedicated one for deleting purposes.